src/sentry/lang/native/processing.py

```diff
@@ -40,6 +40,7 @@
 APPLECRASHREPORT_ATTACHMENT_TYPE = "event.applecrashreport"
 
 
+# TODO @athena: remove this comment before merging
```

Comment:
This method doesn't treat `native` any differently than other platforms. Only has exceptions about `csharp` so I don't think I need to change anything here.